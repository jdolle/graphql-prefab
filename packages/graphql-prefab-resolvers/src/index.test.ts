import { compile } from './index'

jest.mock('./lib/readResolversDir', () =>
  jest.fn().mockReturnValue([ './example' ]),
)

jest.mock('./lib/parseConfig', () =>
  jest.fn().mockReturnValue({ typeName: 'foo', fields: {} }),
)

it('#compile', () => {
  expect(compile()).toMatchSnapshot()
})
