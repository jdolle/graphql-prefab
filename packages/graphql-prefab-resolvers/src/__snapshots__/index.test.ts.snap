// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#compile 1`] = `
Object {
  "foo": Object {},
}
`;
